Products/CMFPlone/interfaces/controlpanel.py

```diff
@@ -39,7 +39,7 @@
 Disallow: /*folder_summary_view$
 Disallow: /*login_form$
 Disallow: /*mail_password_form$
-Disallow: /*search
+Disallow: /@@search
 Disallow: /*search_rss$
```

Comment:
I don't think so, because AFAIK all references are to the view are using `@@`; also this is only a list of things to include by default, anybody can make changes to it later.

I saw the `sitemap` view is also available everywhere and I think that's a mistake.